tests/integration/payments.test.ts

```diff
@@ -121,4 +119,56 @@ describe('payments', () => {
     expect(payments.length).toEqual(2);
     expect(payments[0].id).toEqual(nextPageCursor);
   });
+
+  it('should capture a payment', async () => {
+    const payments = await mollieClient.payments.page();
+
+    const existingPayment = payments.find(payment => payment.metadata == 'capture-test' && payment.status == PaymentStatus.authorized);
```

Comment:
Let's restructure this a bit, so that we prevent creating a new payment each time the test runs. Specifically, if an open payment exists, log the checkout URL for that.

Only create a new payment if no payment exists that can manually be "paid".